Dockerfile

```diff
@@ -9,76 +9,18 @@ COPY composer.json /usr/local/src/
 RUN composer install --ignore-platform-reqs --optimize-autoloader \
     --no-plugins --no-scripts --prefer-dist
 
-# Prepare generic compiler
-FROM php:8.3.7-cli-alpine3.19 as compile
-
-ENV PHP_SWOOLE_VERSION=v5.1.2
-
-RUN \
-  apk add --no-cache --virtual .deps \
-  make \
-  automake \
-  autoconf \
-  gcc \
-  g++ \
-  git \
-  linux-headers \
-  openssl-dev \
-  curl-dev
-  
-RUN docker-php-ext-install sockets
-
-# Compile Swoole
-FROM compile AS swoole
-RUN \
-  git clone --depth 1 --branch $PHP_SWOOLE_VERSION https://github.com/swoole/swoole-src.git && \
-  cd swoole-src && \
-  phpize && \
-  ./configure --enable-sockets --enable-http2 --enable-openssl --enable-swoole-curl && \
-  make && make install && \
-  cd ..
-
 # Executor
-FROM php:8.3.7-cli-alpine3.19 as final
+FROM open-runtimes/base:0.1.0 as final
 
 ARG OPR_EXECUTOR_VERSION
 ENV OPR_EXECUTOR_VERSION=$OPR_EXECUTOR_VERSION
 
-LABEL maintainer="team@appwrite.io"
-
-ENV DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
-ENV DOCKER_API_VERSION=1.43
-
-RUN \
-  apk update \
-  && apk add --no-cache --virtual .deps \
-  make \
-  automake \
-  autoconf \
-  curl-dev \
-  gcc \
-  g++ \
-  linux-headers \
-  && apk add --no-cache \
-  libstdc++ \
-  docker-cli \
-  && docker-php-ext-install sockets \
-  && apk del .deps \
-  && rm -rf /var/cache/apk/*
-
-WORKDIR /usr/local/
-
 # Source code
 COPY ./app /usr/local/app
 COPY ./src /usr/local/src
 
 # Extensions and libraries
 COPY --from=composer /usr/local/src/vendor /usr/local/vendor
-COPY --from=swoole /usr/local/lib/php/extensions/no-debug-non-zts-20230831/swoole.so /usr/local/lib/php/extensions/no-debug-non-zts-20230831/
-
-RUN echo extension=swoole.so >> /usr/local/etc/php/conf.d/swoole.ini
-
-EXPOSE 80
```

Comment:
Don't we need this `EXPOSE 80`?

Reply:
Base image defines that: https://github.com/open-runtimes/docker-base/pull/1/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R62

Plus tests pass, makes me confident it works